examples/terraform/README.md

```diff
@@ -0,0 +1,95 @@
+## How-To: using Terraform to manage an EKS Anywhere (EKS-A) Cluster
```

Comment:
```suggestion
## Using Terraform to manage an EKS Anywhere Cluster
```
I'll try to apply our docs style to my editing here. We try to avoid "EKS-A" in the docs and only spell it out unless it is some literal value.